packages/mjml-core/src/helpers/jsonToXML.js

```diff
@@ -1,16 +0,0 @@
-const jsonToXML = ({ tagName, attributes, children, content }) => {
```

Comment:
We really need to keep that as a helper function as I know some tools use this API. And we might as well need it for #2556 to resolve include into an outputfile.

Reply:
no problem, i've restored it and modified `createComponent` to leverage it